kafka/channel/cmd/channel_dispatcher/main.go

```diff
@@ -26,7 +26,7 @@ import (
 	controller "knative.dev/eventing-contrib/kafka/channel/pkg/reconciler/dispatcher"
 )
 
-const component = "kafkachannel_dispatcher"
+const component = "kafkachannel-dispatcher"
```

Comment:
there is references to this `_` version in kafka/docs/metrics.md  -> mind updateing ?

Reply:
ahhh, thank you